### ISSUE TITLE
mysql_client does not have action :create

### ISSUE THREAD

**Issue by ahrib** [chef-cookbooks/mysql#432]
### Cookbook version
8.0

### Chef-client version
Chef Development Kit Version: 0.15.16
chef-client version: 12.11.18

### Platform Details
OSX 10.11.5

### Scenario:
install mysql client

### Steps to Reproduce:
use the following block (straight from docs) results in error

mysql_client 'default' do
  action :create
end

### Expected Result:
kitchen converge to result in no error

### Actual Result:
kitchen converge results in error:

Chef::Exceptions::ValidationFailed
         ----------------------------------
         Option action must be equal to one of: nothing, install, delete!  You passed :create.

**Comment by JairoAlonso25** [comment]
Same here. I'm using kitchen with VirtualBox and Centos 7.2 base.

**Comment by tpetchel** [comment]
@JairoAlonso25 posted similar feedback on Learn Chef: https://learn.chef.io/manage-a-web-app/rhel/configure-mysql/#comment-2790225557.

Note that although version 8.0.0 is the latest, the tutorial uses version 7.0.0. 

Looking at the sources, it appears that the available actions are indeed `:install` and `:delete`: 
https://github.com/chef-cookbooks/mysql/blob/master/libraries/mysql_client_installation_package.rb#L17

One of the maintainers can correct me, but these actions likely changed between major versions. Although I saw nothing of this change in the CHANGELOG.

If you're following the tutorial, I'd recommend either:

1. Specifying version 7.0.0 in your metadata.rb per the [instructions](https://learn.chef.io/manage-a-web-app/rhel/configure-mysql/#step1).
2. Continuing to specify version 8.0.0, but replace `action :create` with `action :install` - see if it works!

We should also update the README to show more up-to-date examples.

**Comment by JairoAlonso25** [comment]
@tpetchel 

Hi,

1. Using mysql version 7.0.0 works good
2. Using version 8.0.0 and `action :install`:
```
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Failed to complete #converge action: [Unable to satisfy constraints on package mysql due to solution constraint (mysql = 7.0.0). Solution constraints that may result in a constraint on mysql: [(awesome_customers_rhel = 0.1.0) -> (mysql ~> 8.0.0)], [(mysql = 7.0.0)]
Demand that cannot be met: (mysql = 7.0.0)
Artifacts for which there are conflicting dependencies: mysql = 7.0.0 -> [(yum-mysql-community >= 0.0.0), (smf >= 0.0.0)]Unable to find a solution for demands: awesome_customers_rhel (0.1.0), chef-sugar (3.4.0), compat_resource (12.10.7), firewall (2.5.2), httpd (0.4.0), mysql (7.0.0), rbac (1.0.3), selinux (0.9.0), smf (2.2.8), yum (3.11.0), yum-mysql-community (0.2.0)] on default-centos-72
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

**Comment by zanjibar** [comment]
Using mysql version 7.2.0 works good

**Comment by tpetchel** [comment]
@JairoAlonso25,

Re: moving to 8.0.0 - I was able to get the same result as you. I needed to delete my `Berksfile.lock` to regenerate the dependency tree. 

After doing so, I get this error when I run the cookbook when I converge on CentOS 7.2. Do you get the same result?

Perhaps one of the maintainers can weigh in.

```
             Chef::Exceptions::Package
             -------------------------
             No candidate version available for mysql-community-client

             Cookbook Trace:
             ---------------
             /tmp/kitchen/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:41:in `run_action'
             /tmp/kitchen/cache/cookbooks/compat_resource/files/lib/chef_compat/monkeypatches/chef/runner.rb:41:in `run_action'

             Resource Declaration:
             ---------------------
             # In /tmp/kitchen/cache/cookbooks/mysql/libraries/mysql_client_installation_package.rb

       18:       package package_name do
       19:         version package_version if package_version
       20:         options package_options if package_options
       21:         action :install
       22:       end
       23:     end

             Compiled Resource:
             ------------------
             # Declared in /tmp/kitchen/cache/cookbooks/mysql/libraries/mysql_client_installation_package.rb:18:in `block in <class:MysqlClientInstallationPackage>'
```

**Comment by JairoAlonso25** [comment]
@tpetchel Exactly the same result. Maybe it ask for some missing dependency. Centos 7 uses mariadb instead of mysql, but a I can't find a proper solution in documentation.

**Comment by tas50** [comment]
This is fixed in 8.0.1. Sorry about that
